src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/model/Organization.kt

```diff
@@ -19,5 +19,6 @@ data class Organization (
     val industryCode: String? = null,
     val sectorCode: String? = null,
     val prefLabel: PrefLabel? = null,
+    val orgStatus: PrefLabel? = null,
```

Comment:
orgStatus bør være enum, og ikke PrefLabel. Status => label kan utledes

Reply:
Om vi endrer til enum her i DTO'en så blir det forskjellig ut fra hvordan jeg forstod @stigbd ville ha det, men igjen er ikke store forskjellen for min del.

Hva vil du ha Stig?
json med enum:
```
{
  "orgStatus": "Liquidated"
} , {
  "orgStatus": "Normal activity"
}
```

eller jsom med prefLabel:
```
{
  "orgStatus": {
    "nb": "Normal aktivitet",
    "nn": "Normal aktivitet",
    "en": "Normal activity"
  }
} , {
  "orgStatus": {
    "nb": "Avviklet",
    "nn": "Avvikla",
    "en": "Liquidated"
  }
}
```

Reply:
Eg bryr meg ikkje så mykje om enum eller ikkje. Eg vil at RDF blir korrekt

Korrekt: at property orgStatus peiker på ein instans av skos:Concept. (Og denne instansen må vere tilgjengeleg på internettet, enten som BNode, node som er med i grafen, eller node som ligg ein annan stad.)

Reply:
Ok da dropper jeg prefLabel i json og xml